### ISSUE TITLE
Deprecated? 

### ISSUE THREAD

**Issue by okdistribute** [datproject/dat-http#4]
Is this repo still relevant?

**Comment by joehand** [comment]
Yes, still may be useful? It may be duplicated by work in https://github.com/random-access-storage/random-access-http though.

Related: https://github.com/datproject/dat-node/issues/169
